lib/WP_Auth0_LoginManager.php

```diff
@@ -774,6 +772,28 @@ protected function die_on_login( $msg = '', $code = 0, $login_link = true ) {
 		);
 	}
 
+	/*
+	 *
+	 * DEPRECATED
```

Comment:
why this flying on its own away from a valid method?

Reply:
It's not, just a section indicator, everything below is deprecated